app/src/plugins/remoteinput/remoteinput.cpp

```diff
@@ -0,0 +1,73 @@
+#include "remoteinput.h"
+#include <QtPlugin>
+#include <QMap>
+#include <sailfishconnect/device.h>
+
+QMap<QString, int> specialKeysMap = {
+    //0,              // Invalid
+    {"backspace", 1},
+    {"tab", 2},
+    //XK_Linefeed,    // 3
+    {"left", 4},
+    {"up", 5},
+    {"right", 6},
+    {"down", 7},
+    {"pgup", 8},
+    {"pgdn", 9},
+    {"home", 10},
+    {"end", 11},
+    {"enter", 12},
+    {"del", 13},
+    {"esc", 14},
+    //{Qt::Key_SysReq, 15},
+    //{Qt::Key_ScrollLock, 16},
+    //0,              // 17
+    //0,              // 18
+    //0,              // 19
+    //0,              // 20
+    //{Qt::Key_F1, 21},
+    //{Qt::Key_F2, 22},
+    //{Qt::Key_F3, 23},
+    //{Qt::Key_F4, 24},
+    //{Qt::Key_F5, 25},
+    //{Qt::Key_F6, 26},
+    //{Qt::Key_F7, 27},
+    //{Qt::Key_F8, 28},
+    //{Qt::Key_F9, 29},
+    //{Qt::Key_F10, 30},
+    //{Qt::Key_F11, 31},
+    //{Qt::Key_F12, 32},
+};
+
+SailfishConnect::RemoteInputPlugin::RemoteInputPlugin(Device *device, const QString &name, const QSet<QString> &outgoingCapabilities) : KdeConnectPlugin(device, name, outgoingCapabilities)
```

Comment:
Please do not create lines with more than 80 charaters in C++ files. You can configure this limit in QtCreator. Please look through all your code to fix this.